src/utils/sort.js

```diff
@@ -0,0 +1,41 @@
+import { compareDesc } from 'date-fns';
```

Comment:
This package is not in package.json. We will need a `yarn add date-fns@2.0.0-alpha.9`.